src/utils/isSameDay.js

```diff
@@ -2,8 +2,5 @@ import moment from 'moment';
 
 export default function isSameDay(a, b) {
   if (!moment.isMoment(a) || !moment.isMoment(b)) return false;
-  // Speeding up by comparing most likely to differ unit first
-  return a.day() === b.day() &&
```

Comment:
Moment's day of month function is [`.date()`](http://momentjs.com/docs/#/get-set/date/) 😓